spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -358,6 +392,20 @@
               flow_path: 'standard',
             )
 
+            expect(@irs_attempts_api_tracker).to receive(:track_event).with(
+              :idv_document_upload_submitted,
+              success: true,
```

Comment:
I think I'm questioning existing logic, not your code, but -- it's valid to have this many fields blank?